packages/components/src/Typeahead/Typeahead.component.renderers.js

```diff
@@ -13,8 +13,12 @@ export function renderInputComponent(props) {
 	const { caret, key, debounceMinLength, debounceTimeout, icon, inputRef, ...rest } = props;
 
 	const hasIcon = icon || caret;
+	const typeaheadContainerIconClasses = classNames(
```

Comment:
Well I agree for the first point so I restored the `tc-typeahead-typeahead-input-icon` class.

However, it feels to me that it is a bug to add all the time `theme['typeahead-input-icon']` theme class even if you don't have an icon displayed...

Reply:
I agree it looks strange, but I wonder if it will affect this: https://github.com/Talend/ui/blob/master/packages/components/src/Typeahead/Typeahead.scss#L30

I don't know what the heartbeat does exactly, but it will no longer apply to the component if `hasIcon` is false. I don't know if this is OK or not?

Reply:
Well, not sure too... Otherwise, I can replace https://github.com/Talend/ui/blob/master/packages/components/src/Typeahead/Typeahead.scss#L30 by `.tc-typeahead-typeahead-input-icon` WDYT?

Reply:
Well, decided to rename `typeahead-input-icon` to `typeahead-input-container` to preserve compatibility thx